Libraries/rocSPARSE/level_2/bsrmv/main.cpp

```diff
@@ -145,7 +146,7 @@ int main()
                                         &beta,
                                         d_y));
 
-    // 5. Copy y to host from device
```

Comment:
Here and below, I'd suggest creating a separate PR dedicated to `bsr(x)mv` fixes.

Reply:
Done!